zabbixprovisioner/provisioner/prometheus.go

```diff
@@ -9,47 +9,31 @@ import (
 
 type PrometheusAlertRules struct {
 	Groups []struct {
-		Rules []struct {
-			Name        string            `yaml:"alert"`
-			Annotations map[string]string `yaml:"annotations"`
-		} `yaml:"rules"`
+		Rules []PrometheusRule `yaml:"rules"`
```

Comment:
Isn't the program easier to understand when you have one struct to save all data in, than two?

Reply:
We are using `PrometheusRule` in other places in the code, this simplifies the code as you don't have to re map essentialy same types.